### ISSUE TITLE
points not displayed with lower resolution 

### ISSUE THREAD

**Issue by Aphyllanthes** [lukakern/GisPlus#10]
resolution of 2, 3 or 5 resulted to a few or all points not displayed in raster

**Comment by lukakern** [comment]
For me it works fine even with a resolution of 0.5. They are sometimes hard to recognise because of low grey values. Could this be the case? The value for every geometry can be seen in the output.

**Comment by lukakern** [comment]
Yes, we should set a limit to 1 in this case. Maybe we should limit the resolution in general between 0 and 1.
